src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  margin-right: $card-margin-right;
+  margin-bottom: 46px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    box-sizing: content-box;
+    height: 134px;
+    width: 160px;
+    padding: 0 3px;
+
+  }
+
+  &__name-of-product {
+    width: 166px;
+    font-weight: 500;
+    padding-top: 38px;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    line-height: 14px;
+    padding-top: 4px;
+    font-size: 10px;
+    color: $lightgray;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    padding: 24px 0 16px;
+  }
+
+  &__price-text {
+    line-height: 18px;
+    color: $lightgray;
+  }
+
+  &__price-number {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    width: 100%;
+    height: 40px;
+
+    background-color: $lightblue;
+    color: $btn-text-and-bg-color;
+
+    text-transform: uppercase;
+    font-weight: 700;
+    line-height: 16px;
+
+    border-radius: 5px;
+    border: 1px solid $lightblue;
+
+    transition: background-color 0.8s,
```

Comment:
Would be nice to create variable for transition duration, then we can use it everywhere to make transitions consistent